doozerlib/distgit.py

```diff
@@ -196,7 +199,7 @@ def clone(self, distgits_root_dir, distgit_branch):
                     timeout = str(self.runtime.global_opts['rhpkg_clone_timeout'])
                     rhpkg_clone_depth = int(self.runtime.global_opts.get('rhpkg_clone_depth', '0'))
 
-                    if self.metadata.namespace == 'containers' and self.runtime.cache_dir:
+                    if self.metadata.namespace == 'containers':
```

Comment:
`cache_dir` is enabled on buildvm. When this branch is hit, `git` will be used to clone distgit repos instead of `rhpkg`. On local testing environment, cache dir is usually not used.

I would like to use `git` no matter whether cache dir  is used or not.